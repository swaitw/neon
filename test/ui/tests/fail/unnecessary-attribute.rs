#[neon::export(async)]
async fn async_with_async() {}

fn main() {}
