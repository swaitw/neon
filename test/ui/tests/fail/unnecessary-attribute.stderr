error: `async` attribute should not be used with an `async fn`
 --> tests/fail/unnecessary-attribute.rs:1:16
  |
1 | #[neon::export(async)]
  |                ^^^^^
